agent/xds/testdata/clusters/transparent-proxy-dial-instances-directly.envoy-1-20-x.golden

```diff
@@ -206,7 +206,7 @@
     },
     {
       "@type": "type.googleapis.com/envoy.config.cluster.v3.Cluster",
-      "name": "passthrough~kafka.default.dc1.internal.e5b08d03-bfc3-c870-1833-baddb116e648.consul",
+      "name": "passthrough~kafka.default.dc1.internal.11111111-2222-3333-4444-555555555555.consul",
```

Comment:
These changed because now we're not storing the SNI nor SPIFFE ID in the config snapshot. The trust domain is plugged in from the roots when these clusters are generated.